VirtoCommerce.Storefront/appsettings.json

```diff
@@ -10,7 +10,7 @@
     "Endpoint": {
       "Url": "http://localhost/admin",
       "AppId": "27e0d789f12641049bd0e939185b4fd2",
-      "SecretKey": "34f0a3c12c9dbb59b63b5fece955b7b2b9a3b20f84370cba1524dd5c53503a2e2cb733536ecf7ea1e77319a47084a3a2c9d94d36069a432ecc73b72aeba6ea78",
+      "SecretKey": "e823b8093a9f5acacc9363aa4ac283608c2bd24325cb9c0b1b90730b500857e4da2bd09a1f4145dd0e4ee1f4e945113b9359d5aa5fa647cd595c1746343a6fa8",
```

Comment:
don't ever post your SecretKey. Revert this

Reply:
Fixed